metricbeat/module/vsphere/_meta/config.reference.yml

```diff
@@ -10,3 +12,7 @@
   insecure: false
   # Get custom fields when using virtualmachine metric set. Default false.
   # get_custom_fields: false
+  # Set a maximum query size when retrieving objects, this is typically 256 on vmvsphere 6.5 and later, default 256
+  # max_query_size: 256
+  # Set a maximum metric level to obtain, between 1 and 4, lower numbers are more granular, any number higher than one will hit the database rather than in-memory stats
```

Comment:
```suggestion
  # Set a maximum metric level to obtain, between 1 and 4, lower numbers are more granular, and any number higher than one will hit the database rather than in-memory stats
```

Reply:
fixed.